server/bleep/src/query/execute.rs

```diff
@@ -671,6 +673,8 @@ impl ExecuteQuery for OpenReader {
                         repo_ref: doc.repo_ref.to_owned(),
                         lang: doc.lang.clone(),
                         contents: doc.content.clone(),
+                        size: doc.content.len(),
+                        loc: doc.line_end_indices.len(),
```

Comment:
can this number include only lines of code, i.e. not empty?

Reply:
added a new `sloc` field that measures non-blank lines.